posts/2021-03-13-microprofile-config-2.0.adoc

```diff
@@ -199,27 +213,27 @@ With this, you can retrieve all the useful values with the get methods defined i
 
 [source,java]
 ----
-serverNameConfigValue.getSourceName(); // returns “PropertiesConfigSource[source=file:/<path-to-file>/META-INF/microprofile-config.properties]”
-serverNameConfigValue.getSourceOrnial(); // returns “100”, the default ordinal value for META-INF/microprofile-config.properties
+serverNameConfigValue.getSourceName(); // Returns: PropertiesConfigSource[source=file:/<path-to-file>/META-INF/microprofile-config.properties]
+serverNameConfigValue.getSourceOrdinal(); // Returns: 100 (the default ordinal value for META-INF/microprofile-config.properties)
 ----
 
 [#Property-Expression]
 === Property Expression
-Property Expressions provide a way to set and expand variables in property values using the `${}` syntax. For example, the config property `server.url` defined in the <<example-ConfigSource, example ConfigSource>>  as `\http://${server.host}` will be resolved to `\http://example.org` since `server.host` is defined as `example.org`
+Property Expressions provide a way to set and expand variables in property values using the `${}` syntax. For example, the config property `server.url` defined in the <<example-ConfigSource, example ConfigSource>>  as `\http://${server.host}` will be resolved to `\http://example.org` since `server.host` is defined as `example.org`:
 
 [source,java]
 ----
 @Inject
 @ConfigProperty(name="server.url")
-String url; // retrieves the resolved value of “http://example.org”, or “http://example.test.org” if mp.config.profile is set to “testing”
+String url; // Returns: http://example.org (or http://example.test.org if mp.config.profile is set to “testing”)
 ----
 
-You can also do some funky expressions, defining default values, composed expressions, and multiple expressions, which link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#property-expressions[the spec] covers really well.
+You can also implement some funky expressions such as defining default values, composed expressions, and multiple expressions. link:https://download.eclipse.org/microprofile/microprofile-config-2.0/microprofile-config-spec-2.0.html#property-expressions[The spec] covers these really well.
 
-Previous working configurations might behave differently if the configuration in use contains values with Property Expressions syntax.
+Note: Previously working configurations might now behave differently if the configuration happens to contain values with the Property Expressions syntax (`${}`) in them.
 
 [#Extra-Info]
-== Some extra info for the return journey
+== Some Extra Info For The Return Journey
 
 For the following examples, we'll use a slightly more rogue example ConfigSource (let's call it "example ConfigSource v2"):
```

Comment:
Change,
"For the following examples, we'll use a slightly more rogue example ConfigSource (let's call it "example ConfigSource v2"):"
to,
"For the following examples, we use a slightly more rogue example ConfigSource (let's call it "example ConfigSource v2"):"

Reply:
Not sure about this one. We use an active voice throughout the rest of the blog: "we'll", "you'll", "let's" etc.

Reply:
It's about the future tense, which is generally okay for guides, but not other content.